### PR TITLE
Added all commands that required PDM-managed virtual environment

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Added all commands that required PDM-managed virtual environment

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Configuration Update
- [ ] Bump-up dependent library

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -80,3 +80,5 @@ test-unit = "pdm run make test-unit"
 test-integration = "pdm run make test-integration"
 test-e2e = "pdm run make test-e2e"
 coverage-report = "pdm run make coverage-report"
+check-types = "pdm run make check-types"
+schema-types = "pdm run make schema"
```
Comment (onmete) [inline pyproject.toml:84]: More like `generate-schema` instead of `schema-types`?

Comment (tisnik) [reply pyproject.toml:84]: yup will be better

Comment (tisnik) [reply pyproject.toml]: updated @onmete

**Pull request merged**